Dockerfile

```diff
@@ -37,6 +43,8 @@ WORKDIR /dockermount
 COPY requirements/ /dockermount/requirements/
 RUN python3 -m pip install -r requirements/requirements.txt
 
+RUN python3 -m pip install -r requirements/requirements-server.txt ansible==2.9.15 awscli
+
```

Comment:
What is the purpose of the Dockerfile-Server ? We can install both files in one dockerfile